CHANGELOG.rst

```diff
@@ -2,6 +2,42 @@
 Change history
 ==============
 
+2.13.0
+======
+*October 2, 2025*
+
+**New features**
+
+* [:open-klant:`484`] Update phone number validation rules
+* [:open-klant:`501`] Add new migration script ``migrate_to_v2_phonenumbers`` to also migrate phone numbers (see :ref:`migration_user_docs`)
+* [:open-api-framework:`163`] Integrate ``maykin-common``
+
+  * New template for the landing page
+  * Remove duplicate classes e functions that are already exists in the library
+
+**Bugfixes**
+
+* [:open-klant:`486`] Fix ``default_admin_index.json`` via signal during migration execution
+* [:open-klant:`508`] Fix CSS issues on the landing page
+* [:open-klant:`510`] Add missing ``raw_id_fields`` for PartijAdmin, improving performance on the admin detail page
+
+**Maintenance**
+
+* [:open-api-framework:`85`] Increase ``UWSGI_THREADS`` to 4 to improve concurrency.
```

Comment:
Let's also put this as a warning at the top (Referentielijsten has this already iirc)